src/test/java/org/jenkinsci/plugins/ewm/casc/ConfigAsCodeTest.java

```diff
@@ -0,0 +1,92 @@
+package org.jenkinsci.plugins.ewm.casc;
+
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.ewm.definitions.Disk;
+import org.jenkinsci.plugins.ewm.definitions.DiskPool;
+import org.jenkinsci.plugins.ewm.steps.ExwsAllocateStep;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.*;
+
+import org.jvnet.hudson.test.JenkinsRule;
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Test for Configuration As Code Compatibility.
+ *
+ * @author Martin d'Anjou
+ */
+public class ConfigAsCodeTest {
+
+    @ClassRule public static JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void should_support_configuration_as_code() throws Exception {
+        String config = ConfigAsCodeTest.class.getResource("configuration-as-code.yaml").toString();
+        ConfigurationAsCode.get().configure(config);
+
+        // parse the yaml file and check if they match
+        File file  = new File(ConfigAsCodeTest.class.getResource("configuration-as-code.yaml").getFile());
```

Comment:
no need to read the file in twice, you read it in on line 33

Reply:
When i print the config on line 33 it shows the path to the config rather than the content of the config. So I read it twice because i need the content of the yaml file to call the yaml.load() . And it is strange to me that the configure() accepts a path like "/casc/configure.yaml" to do the configuration. Anyway Martin says i can just use hardcoded values so read the file (or even use the yaml) is not necessary.

Reply:
ok in that case you can extract the first call to `ConfigAsCodeTest.class.getResource("configuration-as-code.yaml")` to a variable and then you would do 
`configAsCodeResource.toString()`
and
` new File(configAsCodeResource.getFile())` on it